main.js

```diff
@@ -0,0 +1,42 @@
+const btn = document.getElementById("btn");
+const closeButton = document.getElementById("close");
+const nav = document.querySelector(".desk-head");
+const list = document.querySelectorAll(".common-li");
+
+const portfolioLink = document.querySelector("#portfolio");
+const portfolio = document.querySelector(".work-section");
+const aboutLink = document.querySelector("#about");
+const about = document.querySelector(".about-me");
+const contactLink = document.querySelector("#contact");
+const contact = document.querySelector(".last-contact");
+
+function closeBtn() {
+  nav.classList.remove("desk-li-menu");
+  nav.style.display = "none";
+}
+
+function openPortfolio() {
+  portfolio.scrollIntoView();
+}
+
+function openAbout() {
+  about.scrollIntoView();
+}
+
+function openContact() {
+  contact.scrollIntoView();
+}
+function navList() {
+  nav.classList.add("desk-li-menu");
+  nav.style.display = "block";
+}
+
+list.forEach((element) => {
+  element.addEventListener("click", closeBtn);
+});
+
+btn.addEventListener("click", navList);
+closeButton.addEventListener("click", closeBtn);
+portfolioLink.addEventListener("click", openPortfolio);
+aboutLink.addEventListener("click", openAbout);
+contactLink.addEventListener("click", openContact);
```

Comment:
Kindly try to follow the Figma design for the design of the menu:
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157015718-d0dfd0cb-e000-41dc-9d00-cdce43030e49.png) | ![image](https://user-images.githubusercontent.com/24703171/157015798-a3a5aafd-9667-47c4-83fc-287c8ceaaaed.png) |
- Kindly increase the `font-size` of each option to match the Figma design.
- Please add the `hr` that separate between each option of menu.